packages/zoe/src/contractSupport/ratio.js

```diff
@@ -141,3 +141,57 @@ export const invertRatio = ratio => {
     ratio.numerator.brand,
   );
 };
+
+export const addRatios = (left, right) => {
+  assertIsRatio(right);
+  assertIsRatio(left);
+  assert(
+    left.numerator.brand === left.denominator.brand &&
+      left.numerator.brand === right.numerator.brand &&
+      left.numerator.brand === right.denominator.brand,
+    X`all brands must match:  ${q(left)} ${q(right)}`,
+  );
+
+  return makeRatio(
+    multiply(left.numerator.value, right.denominator.value) +
```

Comment:
could we use the `natSafeMath.add` for consistency?

Reply:
done